src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -37,6 +37,8 @@ import type {BrowsingContextStorage} from './browsingContextStorage.js';
 import type {CdpTarget} from './cdpTarget.js';
 
 export class BrowsingContextImpl {
+  static loggerPrefix = `${LogType.debug}:browsingContext` as const;
```

Comment:
LOGGER_PREFIX

https://google.github.io/styleguide/tsguide.html#identifiers-constants

Also, maybe make it `static readonly` as well?